Packages/com.unity.inputsystem/Documentation~/ActionAssets.md

```diff
@@ -0,0 +1,135 @@
+# Action Assets
```

Comment:
Let's try for some more consistent naming here. In the docs we call it "Action Asset" now, but in the editor UI we call it "Input Actions". Yes, I get that "Input" may feel redundant in the input system docs and that it is important that this is an asset, but still, using a different wording is very confusing (I had to read the page to make sure it was talking about the same things). And we don't call PhysicMaterials "Physic Material Assets" in our docs either.

Reply:
I guess the problem here is that "Input Actions" also refers to the InputAction class, but the asset actually contains more of that. But then, maybe we should rename the asset (also in the editor and API)? "Input Action Collection" or something?

Reply:
I'm fine calling it something like "Input Action Assets" here. IMO focusing on the fact that these assets *principally* contain actions and ignoring the other stuff is okay.

"Input Action Collection" to me is a bit too generic. Action maps are basically collections of actions, too. And in fact, in the API we have `IInputActionCollection` which in fact both `InputActionAsset` and `InputActionMap` implement.

Reply:
So, yeah, let's use `InputActionAsset` - and not drop the `Input` part in the docs.